test/e2e/e2e_suite_test.go

```diff
@@ -169,17 +169,11 @@ var _ = BeforeSuite(func() {
 
 	// Verify capi components are deployed
 	common.WaitDeployment(kindClient, capiNamespace, capiDeploymentName)
-	go func() {
-		defer GinkgoRecover()
-		watchLogs(capiNamespace, capiDeploymentName, logPath)
-	}()
+	watchLogs(capiNamespace, capiDeploymentName, logPath)
```

Comment:
I just realized that we aren't also doing a `WaitDeployment` or a `watchLogs` on the CABPK or KCP components, I'm thinking we probably should be.

Reply:
Created https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/1493 to address